requirements.txt

```
@@ -1,2 +1,2 @@
 aiohttp>=3.6.0,<3.7.0
-websockets>=6.0,<7.0
\ No newline at end of file
+websockets>=6.0,!=7,!=8.0.0,!=8.0.1,<9
```

To be consistent with websockets versioning:
```suggestion
websockets>=6.0,!=7.0,!=8.0,!=8.0.1,<9.0
```

---

Thanks!